mapswipe_workers/mapswipe_workers/project_types/base/tutorial.py

```diff
@@ -46,6 +49,15 @@ def save_tutorial(self):
                     Project Id is invalid: {self.projectId}"""
             )
 
+        if self.projectType in [ProjectType.FOOTPRINT.value]:
+            # we compress tasks for footprint project type using gzip
+            json_string_tasks = json.dumps(tasks).replace(" ", "").replace("\n", "")
+            compressed_tasks = gzip_str.gzip_str(json_string_tasks)
```

Comment:
Can you refactor these 3 lines with the same ones in `project.py` into a function? Mostly to ensure that the logic is exactly the same everywhere, as we'd break things easily otherwise.

Reply:
good catch. I added a function for this.